lua/entities/gmod_wire_expression2/core/custom/pewpewfunctions.lua

```diff
@@ -455,4 +455,17 @@ e2function entity pewCreate( string bullet, string model, vector pos, angle ang
 	return create(self.player, bullet, model, pos, ang)
 end
 
+
+__e2setcost(0)
+
+e2function number timeSinceLastFired( entity cannon )
+    if not IsValid(cannon) then return 0 end
+        return CurTime() - cannon.LastFired
```

Comment:
Some weird spacing on this line

```suggestion
    return CurTime() - cannon.LastFired
```